### ISSUE TITLE
Data::Compare requires CWD be readable

### ISSUE THREAD

**Issue by tmhall** [DrHyde/perl-modules-Data-Compare#16]
I'm attempting to kick off a script as another user.

However, the Data::Compare module appears to REQUIRE that it be able to chdir to the cwd.

```
$ sudo -u reports perl -MData::Compare -E 'say 1'
Can't cd to /home/mhall: Permission denied
BEGIN failed--compilation aborted.
```

Currently using v1.25
```
$ perl -MData::Compare -E 'say Data::Compare->VERSION'
1.25
```

I would argue that this requirement be removed, and the module fail gracefully if it's unable to look for plugins to add via the CWD.

**Comment by DrHyde** [comment]
You should be good if you upgrade to the latest version, 1.26 fixes this, and 1.27 fixes another bug.

**Comment by tmhall** [comment]
Confirmed.

1.27 addressed the issue, and I've requested ops to update the module.

Thanks,
- Miller
